private-templates-service/client/src/components/Dashboard/styled.tsx

```diff
@@ -31,7 +32,8 @@ export const OuterWindow = styled.div`
   flex-direction: row;
   align-items: stretch;
   justify-content: space-between;
-  height: 100%;
+  height: auto;
```

Comment:
do we need height: auto?

Reply:
yeah we do, the dashboard contents were overflowing out of this container when the browser was resized to be smaller, which was causing the bug